### PR TITLE
add Network Cost Calculation Methodology doc

### LINKED ISSUE #393: Document Network Cost Calclulation Methodology
1. With just the daemonset enabled
2. With the daemonset enabled + CUR enabled
3.  With just the CUR enabled.

### PR DESCRIPTION
**Pull request by keithhand**
Closes #393

### CODE REVIEW

File: network-cost-calculations.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Network Cost Calculation Methodology
+==================
+
+There are two primary concerns when factoring in how Network Costs are calculated: [Cloud Integration](./cloud-integration.md) and the existence of the [Network Costs Daemonset](./network-allocation.md).
```
[the remark this answers is not in the document]
Comment (srpomeroy) [reply network-cost-calculations.md:4]: Links need to be absolute for right now. 
https://github.com/kubecost/docs/blob/main/markdown.md#links

**Comment by srpomeroy** [comment]
As opposed to this being a net new doc, we should add this as a section to the existing network allocation doc here: https://github.com/kubecost/docs/blob/main/network-allocation.md

**Comment by bstuder99** [comment]
Has anyone gone ahead and made a PR for the Network Allocation doc for this content?

**Comment by srpomeroy** [comment]
Contents merged into main network cost doc in #478

**Pull request closed without merging**